apps/condo/domains/ticket/schema/TicketComment.js

```diff
@@ -48,6 +49,12 @@ const TicketComment = new GQLListSchema('TicketComment', {
 
     },
     plugins: [uuided(), versioned(), tracked(), softDeleted(), historical()],
+    hooks: {
+        afterChange: async (requestData) => {
+            /* NOTE: this sends different kinds of notifications on comment create */
+            await handleTicketCommentEvents(requestData)
```

Comment:
handle notifications ?

Reply:
> handle notifications ?

Actually there could be different kinds of actions, not only notifications.